### PR TITLE
Automatically add the ITK Python wrapping to build tree Python path.

### PR DESCRIPTION
**Pull request by thewtex**
This was only tested locally on Linux, but it is also implemented for Mac OSX and Windows.

CC: @cryos @mathturtle

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -57,4 +57,11 @@ find_package(Qt4 REQUIRED)
 include_directories(SYSTEM "${QT_INCLUDES}")
 add_definitions(${QT_DEFINITIONS})
 
+find_package(ITK 4.9)
+if(ITK-FOUND AND NOT ITK_WRAP_PYTHON)
```
Comment (mathturtle) [inline CMakeLists.txt]: Did you mean an underscore in ITK_FOUND?

Comment (mathturtle) [inline CMakeLists.txt]: Also, can we test for the FFTW support as well?

Comment (thewtex) [inline CMakeLists.txt]: > Did you mean an underscore in ITK_FOUND?

Yes, thanks for checking that. I will add it to a follow-up.

> Also, can we test for the FFTW support as well?

Yes, ITK_USE_FFTWF and ITK_USE_FFTWD (for float and double) settings are exported by ITKConfig.cmake.  Are both needed?

**Comment by cryos** [comment]
Let's queue this branch up, and merge after the next release too.

**Comment by cryos** [comment]
I fixed this in #353, closing as it this has now been merged after rebase and some minor changes. Thanks @thewtex

**Pull request closed without merging**